ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -282,6 +288,16 @@ export class WalletDB {
       valueEncoding: NULL_ENCODING,
     })
 
+    this.participantIdentities = this.db.addStore({
+      name: 'pi',
+      keyEncoding: new PrefixEncoding(
+        new BufferEncoding(), // account prefix
+        new StringEncoding(), // participant identifier
```

Comment:
maybe we change this to a Buffer? that will avoid the need to serialize into a string when we add an identity in `addParticipantIdentity` and make the key smaller